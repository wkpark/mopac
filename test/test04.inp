  gnorm=0    AIGIN
 Ethane, showing use of Gaussian Z-matrix input (AIGIN is necessary in this run)
 Check that heat of formation is -19.7220
  C  
  C     1     r21    
  H     2     r32       1     a321   
  H     2     r32       1     a321      3  120   0
  H     2     r32       1     a321      3  240   0
  H     1     r32       2     a321      3  d6321 0
  H     1     r32       2     a321      3 -d6321 0
  H     1     r32       2     a321      3  180   0

     r21        1.5
     r32        1.0
     a321     109.0
     d6321     60.0

