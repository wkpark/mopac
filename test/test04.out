 *******************************************************************************
 ** FRANK J. SEILER RES. LAB., U.S. AIR FORCE ACADEMY, COLO. SPGS., CO. 80840 **
 *******************************************************************************

                                MNDO CALCULATION RESULTS


 *******************************************************************************
 *          MOPAC:  VERSION  6.00               CALC'D. 31-OCT-90               
 *  AIGIN    - GEOMETRY MUST BE IN GAUSSIAN FORMAT
 *   T=      - A TIME OF  3600.0 SECONDS REQUESTED
 *  DUMP=N   - RESTART FILE WRITTEN EVERY  3600.0 SECONDS
 *  GNORM=   - EXIT WHEN GRADIENT NORM DROPS BELOW .000E+00
 ***********************************************************************040BY040
  GNORM=0    AIGIN
 Ethane, showing use of Gaussian Z-matrix input (AIGIN is necessary in this run)
 Check that heat of formation is -19.7220

    ATOM   CHEMICAL  BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL    (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                  NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      C       
      2      C         1.50000  *                                  1
      3      H         1.00000  *     109.00000  *                 2    1
      4      H         1.00000        109.00000     120.00000      2    1    3
      5      H         1.00000        109.00000    -120.00000      2    1    3
      6      H         1.00000        109.00000      60.00000  *   1    2    3
      7      H         1.00000        109.00000     -60.00000      1    2    3
      8      H         1.00000        109.00000     180.00000      1    2    3


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C        0.0000    0.0000    0.0000
     2         C        1.5000    0.0000    0.0000
     3         H        1.8256    0.9455    0.0000
     4         H        1.8256   -0.4728    0.8188
     5         H        1.8256   -0.4728   -0.8188
     6         H       -0.3256    0.4728   -0.8188
     7         H       -0.3256    0.4728    0.8188
     8         H       -0.3256   -0.9455    0.0000
  H: (MNDO):  M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)       
  C: (MNDO):  M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)       


          RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =  7


            INTERATOMIC DISTANCES
0
                  C  1       C  2       H  3       H  4       H  5       H  6
 ------------------------------------------------------------------------------
     C    1   0.000000
     C    2   1.500000   0.000000
     H    3   2.055895   1.000000   0.000000
     H    4   2.055895   1.000000   1.637686   0.000000
     H    5   2.055895   1.000000   1.637686   1.637686   0.000000
     H    6   1.000000   2.055895   2.349764   2.864159   2.349764   0.000000
     H    7   1.000000   2.055895   2.349764   2.349764   2.864159   1.637686
     H    8   1.000000   2.055895   2.864159   2.349764   2.349764   1.637686
0
                  H  7       H  8
 ------------------------------
     H    7   0.000000
     H    8   1.637686   0.000000

  GNORM HAS BEEN SET TOO LOW, RESET TO 0.01
 CYCLE:   1 TIME:   1.42 TIME LEFT:   3597.3 GRAD.:   111.588 HEAT:-18.506771    
 CYCLE:   2 TIME:   1.41 TIME LEFT:   3595.9 GRAD.:     2.091 HEAT:-19.719781    
 CYCLE:   3 TIME:   0.69 TIME LEFT:   3595.2 GRAD.:     0.919 HEAT:-19.721616    
 CYCLE:   4 TIME:   0.69 TIME LEFT:   3594.5 GRAD.:     0.121 HEAT:-19.721996    
 CYCLE:   5 TIME:   1.00 TIME LEFT:   3593.5 GRAD.:     0.014 HEAT:-19.722001    
 TEST ON GRADIENT SATISFIED
 PETERS TEST SATISFIED 

 -------------------------------------------------------------------------------
  GNORM=0    AIGIN
 Ethane, showing use of Gaussian Z-matrix input (AIGIN is necessary in this run)
 Check that heat of formation is -19.7220


     PETERS TEST WAS SATISFIED IN BFGS OPTIMIZATION           
     SCF FIELD WAS ACHIEVED                                   


                               MNDO    CALCULATION
                                                       VERSION  6.00
                                                       31-OCT-90               




          FINAL HEAT OF FORMATION =        -19.72200 KCAL


          TOTAL ENERGY            =       -341.67064 EV
          ELECTRONIC ENERGY       =       -935.73200 EV
          CORE-CORE REPULSION     =        594.06135 EV

          IONIZATION POTENTIAL    =         12.69898
          NO. OF FILLED LEVELS    =          7
          MOLECULAR WEIGHT        =     30.069


          SCF CALCULATIONS  =               11
          COMPUTATION TIME =   7.660 SECONDS





    ATOM   CHEMICAL  BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL    (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                  NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      C       
      2      C         1.52090  *                                  1
      3      H         1.10918  *     111.19096  *                 2    1
      4      H         1.10918        111.19096     120.00000      2    1    3
      5      H         1.10918        111.19096    -120.00000      2    1    3
      6      H         1.10918        111.19096      60.00233  *   1    2    3
      7      H         1.10918        111.19096     -60.00233      1    2    3
      8      H         1.10918        111.19096     180.00000      1    2    3


            INTERATOMIC DISTANCES
0
                  C  1       C  2       H  3       H  4       H  5       H  6
 ------------------------------------------------------------------------------
     C    1   0.000000
     C    2   1.520900   0.000000
     H    3   2.182429   1.109177   0.000000
     H    4   2.182429   1.109177   1.791245   0.000000
     H    5   2.182429   1.109177   1.791245   1.791245   0.000000
     H    6   1.109177   2.182429   2.542622   3.110210   2.542592   0.000000
     H    7   1.109177   2.182429   2.542622   2.542592   3.110210   1.791287
     H    8   1.109177   2.182429   3.110210   2.542607   2.542607   1.791224
0
                  H  7       H  8
 ------------------------------
     H    7   0.000000
     H    8   1.791224   0.000000


                  EIGENVALUES

 -35.24055 -24.81402 -15.08649 -15.08633 -13.32762 -12.69911 -12.69898   3.75535
   3.75544   4.37517   4.70782   4.70791   5.77965   5.83586


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

         ATOM NO.   TYPE          CHARGE        ATOM  ELECTRON DENSITY
           1          C           0.0166          3.9834
           2          C           0.0166          3.9834
           3          H          -0.0055          1.0055
           4          H          -0.0055          1.0055
           5          H          -0.0055          1.0055
           6          H          -0.0055          1.0055
           7          H          -0.0055          1.0055
           8          H          -0.0056          1.0056
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID         0.000     0.000     0.000     0.000
 SUM            0.000     0.000     0.000     0.000


          CARTESIAN COORDINATES 

    NO.       ATOM               X         Y         Z

     1         C                  0.0000    0.0000    0.0000
     2         C                  1.5209    0.0000    0.0000
     3         H                  1.9218    1.0342    0.0000
     4         H                  1.9218   -0.5171    0.8956
     5         H                  1.9218   -0.5171   -0.8956
     6         H                 -0.4009    0.5171   -0.8956
     7         H                 -0.4009    0.5171    0.8956
     8         H                 -0.4009   -1.0342    0.0000


          ATOMIC ORBITAL ELECTRON POPULATIONS

   1.21965   0.92864   0.91753   0.91753   1.21965   0.92864   0.91753   0.91753
   1.00555   1.00555   1.00555   1.00554   1.00554   1.00555
  CHANNEL 12 OPENED



 TOTAL CPU TIME:             8.03 SECONDS

 == MOPAC DONE ==
